src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.spec.ts

```diff
@@ -28,4 +36,89 @@ describe('InstructorHomePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with one course without feedback session', () => {
+    const instructorName: string = '';
+    const courseTabModels: any = {
+      course: {
+        courseId: 'CS3281',
+        courseName: 'Thematic Systems',
+        creationDate: '26 Feb 23:59 PM',
+        timeZone: 'Asia/Singapore',
+      },
+      instructorPrivilege: {
+        canModifyCourse: true,
+        canModifySession: true,
+        canModifyStudent: true,
+        canSubmitSessionInSections: true,
+      },
+      sessionsTableRowModels: [],
+      sessionsTableRowModelsSortBy: SortBy.NONE,
+      sessionsTableRowModelsSortOrder: SortOrder.ASC,
+      hasPopulated: true,
```

Comment:
Same for these. Either mix and match between the two existing test cases, if you can cover all bases using just this, or write more test cases.